packages/image_picker/image_picker_for_web/README.md

```diff
@@ -65,8 +68,8 @@ should add it to your `pubspec.yaml` as usual.
 
 You should be able to use `package:image_picker` _almost_ as normal.
 
-Once the user has picked a file, the returned `PickedFile` instance will contain a
-`network`-accessible URL (pointing to a location within the browser).
+Once the user has picked a file, the returned `XFile` instance will contain a
+`network`-accessible Blob URL (pointing to a location within the browser).
```

Comment:
Nit: Should that be `Blob`?

Reply:
Yeah, MDN uses [both](https://developer.mozilla.org/en-US/docs/Web/API/Blob), but the non-monospace version looks like exceptions to the rule.